extensions/2.0/Khronos/KHR_texture_basisu/README.md

```diff
@@ -0,0 +1,84 @@
+# KHR_texture_basisu
+
+## Contributors
+
+* Alexey Knyazev
+* Members of 3D Formats Working Group
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec. Requires `KHR_image_ktx2` extension.
+
+## Overview
+
+**TBD**
+
+## glTF Schema Updates
+
+The `KHR_texture_basisu` extension is added to the `textures` node and specifies a `source` property that points to the index of the `images` node which defines a reference to the KTX2 image with Basis Universal supercompression.
+
+The following glTF will load `image.ktx2` in clients that support this extension, and otherwise fall back to `image.png`.
```

Comment:
(just clarifying) I am assuming that in absence of "source" property on the original texture JSON element, the loaders that support the extension *must* use the image specified through the extension metadata?